test/unit/govuk_index/indexable_content_sanitiser_test.rb

```diff
@@ -0,0 +1,97 @@
+require 'test_helper'
+
+class GovukIndex::IndexableContentSanitiserTest < Minitest::Test
+  def setup
+    super
+    stub_empty_popularity_lookup
+  end
+
+  def test_strips_html_tags_from_indexable_content
+    payload = {
+      "base_path" => "/some/path",
+      "document_type" => "aaib_report",
+      "details" => {
+        "body" => [
+          { "content_type" => "text/govspeak", "content" => "**hello**" },
+          { "content_type" => "text/html", "content" => "<p><strong>hello</strong></p> <a href='www.gov.uk'>marmaduke</a>" }
+        ]
+      }
+    }
+
+    presenter = elasticsearch_presenter(payload)
+
+    assert_equal "\nhello\n marmaduke", presenter.document[:indexable_content]
+  end
+
+  def test_strips_html_tags_from_single_string_item
+    payload = {
+      "base_path" => "/some/path",
+      "document_type" => "aaib_report",
+      "details" => { "body" =>  "hello marmaduke"}
```

Comment:
Should this have some HTML tags in it? Or is this a test of a body without HTML? In which case the test name is a bit confusing.